backend/src/main/java/com/htecgroup/skynest/filter/CustomAuthenticationFilter.java

```diff
@@ -69,16 +73,20 @@ protected void successfulAuthentication(
       FilterChain chain,
       Authentication authentication)
       throws IOException, ServletException {
-    String userName = ((User) authentication.getPrincipal()).getUsername();
-    log.info("{} is successfully logged in.", userName);
 
-    Date expiration = new Date(System.currentTimeMillis() + SecurityConstants.EXPIRATION_TIME);
-    Algorithm algorithm = Algorithm.HMAC256(SecurityConstants.TOKEN_SECRET);
+    User user = (User) authentication.getPrincipal();
 
-    String token = JwtTokens.generateToken(userName, expiration, algorithm);
+    List<String> authorities =
```

Comment:
This creation od authorities list is a good candidate to be in a separate method / Class.

Reply:
We should probably refactor that and the JWT stuff at the same time

Reply:
yeah, leaving //todo in the code is completely fine and useful for future reference